core/src/main/java/org/elasticsearch/common/util/CollectionUtils.java

```diff
@@ -458,4 +474,9 @@ public static int sortAndDedup(final BytesRefArray bytes, final int[] indices) {
         return result;
     }
 
+    public static <E> void addAll(Collection<E> collection, Iterable<ObjectCursor<E>> iterable) {
```

Comment:
Certainly. I added it thinking "I'm sure I'll need this a few times" and then forgot about it.